styles.css

```diff
@@ -4,513 +4,1139 @@
-  font-weight: 400;
-  font-size: 36px;
-  line-height: 44px;
-}
+  .card-1,
+  .card-2,
+  .card-3,
+  .card-4,
+  .card-5,
+  .card-6 {
+    background-image: url("./Img/bg-img.svg");
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
 
-.card-1 p,
-.card-2 p,
-.card-3 p,
-.card-4 p,
-.card-5 p,
-.card-6 p {
-  color: #fff;
-  font-family: arial, sans-serif, 'Inter';
-  font-weight: 400;
-  font-size: 16px;
-  width: 100%;
-  padding: 0 24px;
-}
+  .card-1 h2,
+  .card-2 h2,
+  .card-3 h2,
+  .card-4 h2,
+  .card-5 h2,
+  .card-6 h2 {
+    color: #fff;
+    font-family: 'Crete Round', arial, sans-serif;
+    font-weight: 400;
+    font-size: 36px;
+    line-height: 44px;
+  }
 
-.card-1-ul,
-.card-2-ul,
-.card-3-ul,
-.card-4-ul,
-.card-5-ul,
-.card-6-ul {
-  display: flex;
-  gap: 12px;
-  padding: 5px 24px;
-}
+  .card-1 p,
+  .card-2 p,
+  .card-3 p,
+  .card-4 p,
+  .card-5 p,
+  .card-6 p {
+    color: #fff;
+    font-family: arial, sans-serif, 'Inter';
+    font-weight: 400;
+    font-size: 16px;
+    width: 100%;
+    padding: 0 24px;
+  }
 
-[class*="html-btn"] {
-  color: #fff;
-  background: rgba(255, 255, 255, 0.24);
-  border: 1px solid #8993a4;
-  width: 56px;
-  height: 40px;
-  font-weight: 500;
-  font-family: arial, sans-serif, 'Inter';
-  font-size: 15px;
-  line-height: 20px;
-  cursor: pointer;
-  margin: 10px 0;
-}
+  .h2-d {
+    display: none;
+  }
 
-[class*="bs-btn"] {
-  background: rgba(255, 255, 255, 0.24);
-  border: 1px solid #8993a4;
-  width: 95px;
-  height: 40px;
-  font-weight: 500;
-  font-family: arial, sans-serif, 'Inter';
-  font-size: 15px;
-  line-height: 20px;
-  color: #fff;
-  cursor: pointer;
-  margin: 10px 0;
-}
+  .card-1-ul,
+  .card-2-ul,
+  .card-3-ul,
+  .card-4-ul,
+  .card-5-ul,
+  .card-6-ul {
+    display: flex;
+    gap: 12px;
+    padding: 5px 24px;
+  }
 
-[class*="ruby-btn"] {
-  background: rgba(255, 255, 255, 0.24);
-  border: 1px solid #8993a4;
-  width: 61px;
-  height: 40px;
-  font-weight: 500;
-  font-family: arial, sans-serif, 'Inter';
-  font-size: 15px;
-  line-height: 20px;
-  color: #fff;
-  cursor: pointer;
-  margin: 10px 0;
-}
+  [class*="html-btn"] {
+    color: #fff;
+    background: rgba(255, 255, 255, 0.24);
+    border: 1px solid #8993a4;
+    width: 56px;
+    height: 40px;
+    font-weight: 500;
+    font-family: arial, sans-serif, 'Inter';
+    font-size: 15px;
+    line-height: 20px;
+    cursor: pointer;
+    margin: 10px 0;
+  }
 
-[class*="see-pro"] {
-  color: #fff;
-  font-weight: 700;
-  font-size: 17px;
-  width: 100%;
-  height: 48px;
-  background-color: #ff6b00;
-  border: none;
-  cursor: pointer;
-}
+  [class*="bs-btn"] {
+    background: rgba(255, 255, 255, 0.24);
+    border: 1px solid #8993a4;
+    width: 95px;
+    height: 40px;
+    font-weight: 500;
+    font-family: arial, sans-serif, 'Inter';
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+    cursor: pointer;
+    margin: 10px 0;
+  }
 
-.about-me {
```

Comment:
- [ ] Kindly fix the about me section by using the same font family used in the design
- [ ] Figma design
<img width="1440" alt="Screenshot 2023-02-23 at 13 23 17" src="https://user-images.githubusercontent.com/81444338/221035229-8639cd9a-dec3-4b83-bc6c-95527cec2f6b.png">

- [ ] Your design
<img width="1440" alt="Screenshot 2023-02-23 at 13 22 55" src="https://user-images.githubusercontent.com/81444338/221035352-00a797db-8e2d-47ff-9623-18ea6387b0f1.png">